app/models/device_story.rb

```diff
@@ -3,4 +3,6 @@
 class DeviceStory < ApplicationRecord
   validates :device_urn, uniqueness: true
   validates :device_urn, :custodian_name, presence: true
+  has_many  :device_story_comments
+  accepts_nested_attributes_for :device_story_comments
```

Comment:
This doesn't seem like the right move to me. I'd probably opt to have the device story comment be its own controller (a nested route could be good so you always have the device story id) and write directly.

Could be that's my old-school rails background talking though. Maybe @sasharevzin or @eitoball would be able to cross check me.

Reply:
Ok, will work on separating them.

Reply:
I agree with @matschaffer. We should use [Nested resources](https://guides.rubyonrails.org/routing.html#nested-resources).